### PR TITLE
[ADD] hr_attendance_modification_tracking

### PR DESCRIPTION
**Pull request by jarroyomorales**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
The module name should be ` hr_attendance_modification_tracking` for being more verbose and using nouns, not verbs.

**Comment by HviorForgeFlow** [comment]
What is the benefit of registering attendances on chatter. At first sight, it doesn't make me any sense

**Comment by etobella** [comment]
Well, it is interesting to see who made a change and add the option to add a reason.
If I entered at 10.00 and change it to 8.00 no one can see the change.

**Review by HviorForgeFlow — APPROVED** [review]
Good point :smile: 
Code + functional review
LGTM :+1:

File: hr_attendance_modification_tracking/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Hr Attendance Modification Tracking',
+    'summary': """
+        Attendance changes will now be registered in the chatter.""",
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Creu Blanca,Odoo Community Association (OCA)',
+    'website': 'www.creublanca.es',
```
Comment (nikul-serpentcs) [inline hr_attendance_modification_tracking/__manifest__.py]: @jarroyomorales URL should be `https://github.com/OCA/hr`

File: hr_attendance_modification_tracking/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    'depends': [
+        'hr_attendance',
+        'mail',
+    ],
+    'data': [
+        'views/hr_attendance_view.xml',
+    ],
+    'demo': [
```
Comment (nikul-serpentcs) [inline hr_attendance_modification_tracking/__manifest__.py]: if no file the remove `key`

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Minor Change

**Comment by etobella** [comment]
@nikul-serpentcs Your comments have been attended. Can you review it again?

**Review by etobella — APPROVED** [review]
LGTM :+1:

**Comment by jarroyomorales** [comment]
@nikul-serpentcs Is it right now for you?

**Comment by JordiBForgeFlow** [comment]
@nikul-serpentcs please in your reviews only mark 'request changes' when the changes are blocking. Otherwise just put a comment like 'small reviews'. We see that your reviews can often block the final approval, specially if you don't follow up quickly on the update of your review.

**Comment by nikul-serpentcs** [comment]
> @nikul-serpentcs please in your reviews only mark 'request changes' when the changes are blocking. Otherwise just put a comment like 'small reviews'. We see that your reviews can often block the final approval, specially if you don't follow up quickly on the update of your review.

Yes Sure @jbeficent

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Pull request merged**